marklogic-data-hub/src/main/resources/ml-modules/services/collector.xqy

```diff
@@ -31,9 +34,11 @@ declare option xdmp:mapping "false";
 declare function service:build-response($items as item()*)
   as document-node()
 {
-  document {
-    json:to-array($items) ! xdmp:to-json(.)
-  }
+  perflog:logit('Collector.buildResponse',function() {
```

Comment:
I think wrapping this function in the perflog is a bit overkill. It's already covered by the functions below. The idea is to get a sense of how long a transaction is taking. Thus, I'd prefer to see 1 log per transaction.